api_docs/deploying-bots.md

```diff
@@ -216,13 +229,17 @@ running it manually.
     The standard output of the Botserver will be logged to the path in
     your *supervisord* configuration.
 
+{end_tabs}
+
 If you are hosting the Botserver yourself (as opposed to using a
 hosting service that provides SSL), we recommend securing your
 Botserver with SSL using an `nginx` or `Apache` reverse proxy and
 [Certbot](https://certbot.eff.org/).
 
 ### Troubleshooting
 
+{start_tabs}
```

Comment:
These are also not instructions, so maybe should be formatted in another way.

Reply:
Hmm, maybe as bullet points instead?

![image](https://github.com/zulip/zulip/assets/2343554/17b019db-dc64-470d-b628-dc4c984569bd)

Reply:
That second bullet point looks odd. I think the second sentence should either be in the same paragraph or there should be a paragraph break.